test/unit/controller/predicates/is_intersection_layer.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
```

Comment:
This file is full of good tests and is pretty comprehensive. Very nice :)

Reply:
@orangejulius Thanks!